### PR TITLE
Update 3.4.0 release notes

### PR DESCRIPTION
**Pull request by irhkang**

### CODE REVIEW

File: docs/release_notes.rst [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+v3.4.0
+------
+
+:date: 2019-06-13
+
+This is a backwards compatible, minor release.
+
+Changes in this release:
+
+ * Adds a TTLAttribute that dynamo uses to delete records
```
Comment (ikonst) [inline docs/release_notes.rst]:
```suggestion
 * Adds a TTLAttribute that specifies when items expire
```

File: docs/release_notes.rst [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+ * Enables TTL deletion on a dynamo table if the corresponding model has a TTLAttribute
```
Comment (ikonst) [inline docs/release_notes.rst]:
```suggestion
 * Enables time-to-live on a DynamoDB table if the corresponding model has a TTLAttribute
```

File: docs/release_notes.rst [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
+ * default_for_new sets default values only for unsaved objects
```
Comment (ikonst) [inline docs/release_notes.rst]:
```suggestion
 * Adds a default_for_new parameter to Attribute which is a default that applies to new items only
```

Comment (irhkang) [reply docs/release_notes.rst]: Is this too much detail?

**Comment by irhkang** [comment]
@ikonst changed

File: docs/release_notes.rst (hunk 3 above)
Comment (ikonst) [reply docs/release_notes.rst]: I agree, let's ~remove~ the part in parenthesis

**Pull request merged**